userspace/libscap/engine/bpf/bpf.h

```diff
@@ -47,6 +47,11 @@ struct bpf_engine
 	int m_bpf_map_fds[BPF_MAPS_MAX];
 	int m_bpf_prog_array_map_idx;
 	char m_filepath[PATH_MAX];
-};
 
-#define SCAP_HANDLE_T struct bpf_engine
+	/* ELF related */
+	int program_fd;
+	Elf *elf;
+	GElf_Ehdr ehdr;
+
+	interesting_tp_set open_tp_set;
```

Comment:
Store the opening requested tp set, so that a cycle:
* start capture
* stop capture
* start capture

will work, and enable the correct tracepoints as requested during scap_open.